basictracer/util.py

```diff
@@ -17,5 +16,5 @@ def generate_id():
     pid = os.getpid()
     if (_fork_guard_pid == 0) or (_fork_guard_pid != pid):
         _fork_guard_pid = pid
-        guid_rng.jumpahead(int(1000000 * time.time()) ^ pid)
+        guid_rng.seed(random.randrange(0, 0xFFFFFFFF))
```

Comment:
Seed is fine, but the arg must incorporate pid (and should incorporate time IMO)